packages/apps/dev-wallet/src/modules/wallet/wallet.service.ts

```diff
@@ -101,7 +101,7 @@ export async function createProfile(
     name: profileName,
     networks,
     secretId,
-    accentColor: idToColor(uuid),
+    accentColor: accentColor || defaultAccentColor,
```

Comment:
I was thinking the other day, due to a conversation with linda, that we could always assign the color for the user